src/components/search-filter/search-filter.stories.js

```diff
@@ -0,0 +1,111 @@
+import { html } from "lit-html";
+import "../button";
+import "../select";
+import "../input-field";
+import "../search-filter";
+import "../link";
+import styles from "./styles.scss";
+import buttonStyles from "../button/styles.scss";
+import inputStyles from "../input-field/styles.scss";
+import formStyles from "../form/styles.scss";
+import selectStyles from "../select/styles.scss";
+import linkStyles from "../link/styles.scss";
+import { stylesheet, docsIntro } from "../../../.storybook/utils.js";
+import { args, argTypes } from "./config";
+
+export default {
+  title: "native-elements/vl-search-filter",
+  decorators: [
+    (story) =>
+      html`${stylesheet(
+        `${styles}${buttonStyles}${inputStyles}${formStyles}${selectStyles}${linkStyles}`
+      )}${story()}`,
+  ],
+  parameters: {
+    docs: {
+      description: {
+        component: docsIntro({
+          stylesheets: ["search-filter"],
+          root: "search-filter",
+          intro: "Search filter component.",
+        }),
+      },
+    },
+  },
+  args: { ...args },
+  argTypes: { ...argTypes },
+};
+
+export const Default = ({ title, alt, mobileModal, mobileModalTitle }) => html`
+  <div
+    is="vl-search-filter"
+    data-vl-title=${title}
+    ?data-vl-alt=${alt}
+    ?data-vl-mobile-modal=${mobileModal}
+    data-vl-mobile-modal-title=${mobileModalTitle}
```

Comment:
Waar zie jij iets gebeuren? 😄

Reply:
Fixed. Was nog een bug in de component van vroeger.

Reply:
Lastig te reproduceren. Ik zou een extra story maken voor de mobile variant dan met de juiste attributen getoggled, en attributen die niks doen in de `default` story mogen gedisabled worden. Als je de mobiele variant showcased in storybook kan je de breedte van je viewport aanpassen per story; https://storybook.js.org/docs/web-components/essentials/viewport

Reply:
Extra story gemaakt met als default viewport mobile.